code/game/gamemodes/shadowling/shadowling_items.dm

```diff
@@ -100,3 +100,18 @@
 		td << "<span class='userdanger'>You enter the rift. Deafening chimes jingle in your ears. You are swallowed in darkness.</span>"
 	playsound(loc, 'sound/effects/EMPulse.ogg', 25, 1)
 	qdel(td)
+
+
+/obj/item/organ/internal/shadowtumor
+	name = "black tumor"
+	icon_state = "blacktumor"
+	origin_tech = "biotech=4"
```

Comment:
Only four? Come on, anything you get from an antag via surgery (alien brain, for example) should unlock the awesome, gated stuff! How about biotech 5?